src/state/session/index.tsx

```diff
@@ -164,8 +164,8 @@ export function Provider({children}: React.PropsWithChildren<{}>) {
         throw new Error(`session: createAccount failed to establish a session`)
       }
 
-      const account: persisted.PersistedAccount = {
-        service,
+      const account: SessionAccount = {
+        service: agent.service.toString(),
```

Comment:
Just using the agent/session as the source of truth for this stuff so that the persisted account never gets out of date